typing/typemod.ml

```diff
@@ -331,6 +335,7 @@ let map_ext fn exts rem =
    making them abstract otherwise. *)
 
 let rec approx_modtype env smty =
+  (* Need to process warning attributes here ?? *)
```

Comment:
Should warnings rather be disabled when computing approximative module types (i.e. in `approx_*` functions) since those approximation are only used to initialize the typing of recursive module types and warnings can be emitted once the full module types have been computed?

Reply:
Well, in #1191, the call to approx_* function is not protected against warnings, but it should probably be if the current PR is merged before.